examples/WebUSB/webusb_serial/webusb_serial.ino

```diff
@@ -49,7 +54,7 @@ void setup()
   Serial.begin(115200);
 
   // wait until device mounted
-  while( !USBDevice.mounted() ) delay(1);
+  //while( !TinyUSBDevice.mounted() ) delay(1);
```

Comment:
did you mean to comment it out?

Reply:
yeah, it is not really needed. I think most of this can be removed from example. Not really needed to wait, leave it as comment as hint for user should they need it.

Reply:
But you are right, this is un-related changes. I just push an update to revert this. Thanks for spotting this, you really got an eagle eye :)